packages/shared/src/features/annotation/types.ts

```diff
@@ -57,3 +58,15 @@ export const UpdateAnnotationScoreData = z.discriminatedUnion("dataType", [
   UpdateAnnotationScoreBase.merge(CategoricalData),
   UpdateAnnotationScoreBase.merge(BooleanData),
 ]);
+
+// annotation queues
+
+export const CreateQueueData = z.object({
+  name: z.string().min(1).max(35),
+  description: z.string().max(1000).optional(),
+  scoreConfigs: z.array(z.string()).min(1, {
```

Comment:
is this an array of Ids? If yes, we can improve the naming e.g. scoreConfigIds.

Reply:
Yes it's an array of Ids. Fair, agree on naming